### DISCUSSION TITLE
[Feature] <ability to save a on device album as a Immich album>

### DISCUSSION THREAD

**Discussion by Skywalker8510** [immich-app/immich#6627] [Feature Request]
### The feature

ability to save a on device album as a Immich album. Being able to long press on a on device album and have an option for add as Immich album to make it easy to move albums over

Im on iOS i dont know how the on device albums section works for other platforms

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
